### DISCUSSION TITLE
How can I change the naming rules for generated image files, thank you

### DISCUSSION THREAD

**Discussion by BNP1111** [lllyasviel/Fooocus#2015] [Q&A] [answered]
How can I change the naming rules for generated image files, thank you

**Comment by VintageGenetics** [comment]
Thisis a good question, would be handy when several images are generated for a potential batch save.

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/pull/1932, currently the name is always image png
